### PR TITLE
Enable Perfmon in production

### PR DESCRIPTION
**Pull request by gravityrail**
This PR just enables the "perfmon" performance monitor in production.

It's been running in staging for a few weeks without any issues being reported, so I think this is a safe change.

Given that we're in the process of implementing some new caching features, it would be good to see whether this allows us to capture a baseline of where (and for how long) users are waiting for data to load so that we can evaluate the impact of any improvements.

It's also possible that this doesn't give us good enough data, in which case we can just rip it out :)

This also includes a change which adds a class to the settings for that allows us to detect when it's loading remote values.

cc @rralian @nb

### CODE REVIEW

File: client/my-sites/site-settings/form-general.jsx [hunk 1]
```diff
@@ -373,7 +373,7 @@ module.exports = React.createClass( {
 		}
 
 		return (
-			<div>
+			<div className={ this.state.fetchingSettings ? 'is-loading' : '' }>
```
Comment (rralian) [inline client/my-sites/site-settings/form-general.jsx]: Is this related or did it sneak in somehow?

**Comment by rralian** [comment]
Not sure if the change to form-general.jsx is intended or not, but it seems like it probably doesn't belong in this PR. Or it should be mentioned in the PR description.

I've tested the branch (in chrome, safari, and firefox) with the instructions given in `/client/lib/perfmon/README.md` to better see the events and also looked through `/client/lib/perfmon/index.js` and I don't see any blocker issues. Though `/client/lib/perfmon/index.js` has a lot of linting issues where it's not following our style guidelines. Would be nice to clean that up in a separate PR. This looks :+1: to me. But should probably get this tested on IE as well (I don't have it on this machine).

**Comment by gravityrail** [comment]
@rralian: The other change was intentional, sorry forgot to include in description.

Added this:

> This also includes a change which adds a class to the settings for that allows us to detect when it's loading remote values.

I will merge this and do the cleanup in a separate PR. Thank you for the feedback.
